site-src/geps/gep-713.md

```diff
@@ -0,0 +1,496 @@
+# GEP-713: Policy Attachment
+
+* Issue URL: https://github.com/kubernetes-sigs/gateway-api/issues/713
+* Status: Implementable
+
+## TLDR
+
+This GEP aims to standardize policy attachment to resources associated with
+Gateway API. This will be important for providing a consistent experience across
+implementations of the API, even for configuration details that may not be fully
+portable.
+
+## Goals
+
+* Establish a pattern for policy attachment which will be used for any policies
+  included in the Gateway API spec
+* Establish a pattern for policy attachment which should be used for any
+  implementation specific policies used with Gateway API resources
+* Provide a way to distinguish between required and default values
+* Enable policy attachment at all relevant scopes, including Gateways, Routes,
+  Backends, and maybe Namespaces
+* Provide a means of attachment that works for both ingress and mesh uses of
+  this API
+* Provide a consistent way that both included and implementation-specific
+  policies can be interpreted
+
+## Out of scope
+
+* Define all potential policies that may be attached to resources
+* Design the full structure and configuration of policies
+
+## API
+
+This approach is building on concepts from all of the alternatives discussed
+below. This is very similar to the existing BackendPolicy resource in the API,
+but also borrows some concepts from the ServicePolicy proposal.
```

Comment:
We should have a link to this doc or something so that people can pick up the extra context if they want to.

Reply:
Good idea, added a link